src/components/Modal.js

```diff
@@ -40,34 +40,37 @@ const ModalPanel = styled.div`
   }
 `
 
-const Modal = props => (
-  <ModalWrapper align='center' justify='center'>
-    <ModalBackdrop
-      z={8888}
-      bg='rgba(0,0,0,0.4)'
-      top
-      right
-      bottom
-      left
-      onClick={() => props.cancel()}
-    />
-    <ModalPanel>
-      {!!props.title && (
-        <Text mb={50}>
-          <strong>{props.title}</strong>
-        </Text>
-      )}
-      {props.children}
-      <Flex mt={60} align='center' justify='center'>
-        <Button style={{ marginRight: 20 }} onClick={props.cancel}>
-          Cancel
-        </Button>
-        <Button primary onClick={props.done}>
-          {props.action}
-        </Button>
-      </Flex>
-    </ModalPanel>
-  </ModalWrapper>
-)
+const Modal = props => {
+  const title = props.titleElement || props.title
+  return (
+    <ModalWrapper align='center' justify='center'>
+      <ModalBackdrop
+        z={8888}
+        bg='rgba(0,0,0,0.4)'
+        top
+        right
+        bottom
+        left
+        onClick={() => props.cancel()}
+      />
+      <ModalPanel>
+        {!!title && (
+          <Text mb={50}>
+            <strong>{title}</strong>
```

Comment:
I think we should only be wrapping the `title` in a `<strong>` tag if it is a string